cmd/juju/cloud/list.go

```diff
@@ -166,11 +167,12 @@ func (c *listCloudsCommand) Run(ctxt *cmd.Context) error {
 		}
 		output = clouds
 	default:
-		if c.controllerName == "" && !c.Local {
-			ctxt.Infof(
-				"There are no controllers running.\nYou can bootstrap a new controller using one of these clouds:\n")
-		}
-		if c.controllerName != "" {
+		if c.controllerName == "" {
```

Comment:
-1 on this change as it introduces extra indentation
The original logic was easy enough to read